src/v2/Apps/ViewingRoom/__tests__/ViewingRoomClosedApp.jest.tsx

```diff
@@ -0,0 +1,98 @@
+import React from "react"
+import { MockBoot, renderRelayTree } from "v2/DevTools"
+import ViewingRoomApp from "../ViewingRoomApp"
+import { graphql } from "react-relay"
+import { ViewingRoomClosedApp_Test_QueryRawResponse } from "v2/__generated__/ViewingRoomClosedApp_Test_Query.graphql"
+import { Breakpoint } from "@artsy/palette"
+
+jest.unmock("react-relay")
+jest.mock("v2/Artsy/Router/useRouter", () => ({
+  useRouter: () => ({
+    match: {
+      params: {
+        slug: "subscription-demo-gg-guy-yanai",
+      },
+    },
+  }),
+  useIsRouteActive: () => false,
+}))
+
+describe("ViewingRoomApp", () => {
+  const slug = "subscription-demo-gg-guy-yanai"
+
+  const getWrapper = async (
+    breakpoint: Breakpoint = "lg",
+    response: ViewingRoomClosedApp_Test_QueryRawResponse = ViewingRoomClosedAppFixture
+  ) => {
+    return await renderRelayTree({
+      Component: ({ viewingRoom }) => {
+        return (
+          <MockBoot breakpoint={breakpoint}>
+            <ViewingRoomApp viewingRoom={viewingRoom}>
```

Comment:
Couple things -- it would either 
a) be better if the tests in this file moved into the main test file, duplicating and rendering a component separate from `ViewingRoomClosed`; or 
b) don't move these tests and delete file, and instead render `<ViewingRoomClosed>` here and test against whats in the file, versus rendering the parent and asserting against its children.